### PR TITLE
chore(analytics): resolve swiftformat errors and warnings

### PR DESCRIPTION
**Pull request by phantumcode**
## Description
Resolving SwiftFormat errors and warnings for the Analytics plugin.

## General Checklist

- [ ] Added new tests to cover change, if needed
- [ ] Build succeeds with all target using Swift Package Manager
- [ ] All unit tests pass
- [ ] All integration tests pass
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [ ] PR title conforms to conventional commit style
- [ ] New or updated tests include `Given When Then` inline code documentation and are named accordingly `testThing_condition_expectation()`
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/AWSPinpointAnalyticsPlugin+Configure.swift [hunk 1]
```diff
@@ -82,10 +82,11 @@ extension AWSPinpointAnalyticsPlugin {
     // MARK: Internal
 
     /// Internal configure method to set the properties of the plugin
-    func configure(pinpoint: AWSPinpointBehavior,
+    internal func configure(pinpoint: AWSPinpointBehavior,
                    networkMonitor: NetworkMonitor,
                    globalProperties: AtomicDictionary<String, AnalyticsPropertyValue> = [:],
-                   isEnabled: Bool = true) {
+                   isEnabled: Bool = true)
+    {
```
Comment (ruisebas) [inline AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/AWSPinpointAnalyticsPlugin+Configure.swift:89]: 😖

File: AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Configuration/AWSPinpointAnalyticsPluginConfiguration.swift [hunk 2]
```diff
@@ -54,20 +54,20 @@ public struct AWSPinpointAnalyticsPluginConfiguration {
 
         let pluginConfiguration = try AWSPinpointPluginConfiguration(pinpointAnalyticsConfig)
 
-        let configOptions: AWSPinpointAnalyticsPlugin.Options
-        if let options {
-            configOptions = options
+        let configOptions: AWSPinpointAnalyticsPlugin.Options = if let options {
```
Comment (ruisebas) [inline AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Configuration/AWSPinpointAnalyticsPluginConfiguration.swift:57]: Maybe it's just me, but this new syntax is way more confusing than the previous one.

Comment (phantumcode) [reply AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Configuration/AWSPinpointAnalyticsPluginConfiguration.swift]: agreed, revert/remove conditional assignment rule.

File: AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/AWSPinpointAnalyticsPlugin+Configure.swift (hunk 1 above)
Comment (phantumcode) [reply AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/AWSPinpointAnalyticsPlugin+Configure.swift]: moved brace back to same line.

**Review by ruisebas — APPROVED** [review]

**Pull request merged**